engine.py

```diff
@@ -504,9 +504,53 @@ def save_to_path(self, document, path):
 
             # make the document being processed the active document
             self.adobe.app.activeDocument = document
-
-            # TODO need to use appropriate save options here (jpg, tiff, etc)
-            document.saveAs(self.adobe.File(path))
+        
+            (_, ext) = os.path.splitext(path)
+
+            # valid file extensions and respective save option in photoshop 
```

Comment:
Do any of these options require and non-default settings? I know we set this in the version upload code: https://github.com/shotgunsoftware/tk-photoshopcc/blob/master/hooks/tk-multi-publish2/basic/upload_version.py#L222

Not saying we need to do anything different here, just curious if the default options are sufficient in all cases.

Reply:
You are right, we need to specify the quality (default value is 3).